Source/ALS/Public/AlsAnimationInstance.h

```diff
@@ -335,7 +335,7 @@ inline void UAlsAnimationInstance::MarkPendingUpdate()
 
 inline void UAlsAnimationInstance::MarkTeleported()
 {
-	TeleportedTime = GetWorld()->GetTimeSeconds();
```

Comment:
Should we change the type of `TeleportedTime` to double instead? Or just let it wrap to float type.

Reply:
I think double is better.